src/moveit_commander/planning_scene_interface.py

```diff
@@ -105,6 +105,14 @@ def __make_mesh(self, name, pose, filename, scale = (1, 1, 1)):
         pyassimp.release(scene)
         return co
     
+    def __make_existing(self, name):
+        """
+        Create an empty Collision Object, used when the object already exists 
+        """
+        co = CollisionObject()
+        co.id = name
```

Comment:
No it does not. moveit_commander is quite light compared to the C++ implementation. 

Yoan Mollard 
Research engineer – 3rd hand project 

INRIA Research center, Flowers lab 
http://flowers.inria.fr/ 
http://3rdhandrobot.eu/ 

----- Original Message -----

> From: "Dave Coleman" <notifications@github.com>
> To: "ros-planning/moveit_commander" <moveit_commander@noreply.github.com>
> Cc: "Yoan Mollard" <yoan.mollard@inria.fr>
> Sent: Monday, 30 March, 2015 4:36:23 PM
> Subject: Re: [moveit_commander] Planning scene improvements + added python
> wrapper for MoveGroup.asyncExecute() (#30)

> In src/moveit_commander/planning_scene_interface.py :
> > @@ -105,6 +105,14 @@ def __make_mesh(self, name, pose, filename, scale =
> > (1, 1, 1)):
> >          pyassimp.release(scene)
> >          return co
> >
> > +    def __make_existing(self, name):
> > +        """
> > +        Create an empty Collision Object, used when the object already
> > exists
> > +        """
> > +        co = CollisionObject()
> > +        co.id = name

> i actually do not use moveit_commander but i would assume it already has a
> planning scene monitor initialized that is keeping track of all the planning
> scene objects. does it not?

> —
> Reply to this email directly or view it on GitHub .